### ISSUE TITLE
Suggestion: enforce using @overrides

### ISSUE THREAD

**Issue by dhorkin** [mkorpela/overrides#25]
So, the decorator you've created is pretty cool. However, it doesn't enforce usage.

What I'd like to see, as a slight variation on your example, kinda like the reverse of what `@abstractmethod` in abstract superclass will do if not implemented in subclass

```Python
from overrides import overrides

class SuperClass(EnforceOverrides):

    def method(self):
        """This is the doc for method and will be shown in subclass method too!"""
        return 2

class SubClass(SuperClass):

# error, because the @overrides annotation is missing
    def method(self):
        return 1
```

I'm not 100% sure this is possible in Python (might really require a linting plugin or metaclass?), but it would be really useful for enforcing good code documentation.

**Comment by mkorpela** [comment]
I've implemented this now in https://github.com/mkorpela/overrides/commit/76b0933f1b5df888b36ca43d92238dd7546a1f6a

**Comment by mkorpela** [comment]
Released in 2.4! Thank you for great idea!
